src/components/PhotoPage/FAQSection.js

```diff
@@ -0,0 +1,181 @@
+			borderBottomWidth: 2,
+			borderBottomStyle: 'solid'
+		},
+		'&::before': {
+			display: 'none'
+		},
+		'&$expanded': {
+			margin: 0
+		}
+	},
+	expanded: {}
+}))(MuiAccordion);
+
+const AccordionSummary = withStyles(theme => ({
+	root: {
+		minHeight: 'auto',
+		'&$expanded': {
+			minHeight: 'auto'
+		},
+		[theme.breakpoints.down('xs')]: {
+			paddingLeft: theme.spacing(1),
+			paddingRight: theme.spacing(1)
+		}
+	},
+	content: {
+		margin: '12px 0',
+		'&$expanded': {
+			margin: '12px 0'
+		}
+	},
+	expanded: {}
+}))(MuiAccordionSummary);
+
+const AccordionDetails = withStyles(theme => ({
+	root: {
+		paddingBottom: theme.spacing(6),
+		[theme.breakpoints.down('xs')]: {
+			paddingBottom: theme.spacing(4),
+			paddingLeft: theme.spacing(1),
+			paddingRight: theme.spacing(1)
+		}
+	}
+}))(MuiAccordionDetails);
+
+const useStyles = makeStyles(theme => ({
+	question: {
+		fontWeight: theme.typography.fontWeightMedium,
+		color: theme.palette.secondary.main,
+		fontFamily: theme.typography.fontFamily
+	}
+}));
+
+function AccordionQA({ index, question, answer }) {
+	const [expanded, setExpanded] = useState(index === 0);
+	const classes = useStyles();
+	const panelName = 'faqPanel' + index;
+	const theme = useTheme();
+
+	return (
+		<Accordion
+			key={panelName}
+			square
+			expanded={expanded}
+			onChange={() => setExpanded(e => !e)}>
+			<AccordionSummary
+				expandIcon={expanded ? <Remove /> : <Add />}
+				aria-controls={panelName + '-content'}
+				id={panelName + '-header'}>
+				<Typography variant='body1' className={classes.question}>
+					{question}
+				</Typography>
+			</AccordionSummary>
+			<AccordionDetails>
+				<Typography variant='body1' style={{ maxWidth: theme.breakpoints.values.md * 0.8 }}>
+					{answer}
+				</Typography>
+			</AccordionDetails>
+		</Accordion>
+	);
+}
+
+AccordionQA.propTypes = {
+	question: PropTypes.node.isRequired,
+	answer: PropTypes.node.isRequired,
+	index: PropTypes.number.isRequired
+};
+
+function FAQSection() {
+	const faqs = [
+		{
+			question: `Where do I find the ACM youtube channel?`,
+			answer:
+				<>
+					We post all recordings of our workshops on the ACM youtube channel, which can be found <Link href="https://www.youtube.com/c/ACMUCLA/videos">here</Link>!
+				</>
+		},
+		{
+			question: `How do I participate?`,
+			answer:
+				<>
+					We recommend attending workshops and events (such as HOTH or Hack Kitchen)
+					, joining a Hack Fam, or asking us questions in the ACM discord!
+				</>
+		},
+		{
+			question: `Your past workshops seem interesting, how can I learn more?`,
+			answer:
+				<>
+					Check out the <Link href="https://hack.uclaacm.com/archive">archive</Link> page!
+				</>
+		},
+		{
+			question: `How do I join your team?`,
+			answer:
+				<>
+					We start recruitment for internship positions every fall
+					, following the ACM general meeting (which you should attend)!
+					Every intern becomes an officer following a quarter of training.
```

Comment:
```suggestion
					After a quarter of shadowing officers, interns may have the opportunity to become full-time officers. All of our current officers started as interns and we highly encourage you to look out for applications in the Fall!
```
Or something similar to that 😃 

For "legal" reasons we shouldn't guarantee that all interns become officers. While historically all interns have become officers, this has never actually been a guarantee.